File: Project/1dv600-lab-java/src/main/java/lnu/models/book.java
package lnu.models;

import javax.xml.bind.annotation.*;

@XmlRootElement(name = "book")
@XmlType(propOrder = { "author", "title", "genre", "price", "publish_date", "description" })
public class book {

    private String id;
    private String title;
    private String author;
    private String price;
    private String genre;
    private String publish_date;
    private String description;

    public book() {
        id = "";
        title = "";
        author = "";
        price = "";
        genre = "";
        publish_date = "";
        description = "";
    }

    public book(String id, String title, String author, String description, String genre, String price,
            String publish_date) {
        this.id = id;
        this.title = title;
        this.author = author;
        this.price = price;
        this.genre = genre;
        this.publish_date = publish_date;
        this.description = description;
    }

    public String getId() {
        return id;
    }

    @XmlAttribute
    public void setId(String id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }

    public String getGenre() {
        return genre;
    }

    public void setGenre(String genre) {
        this.genre = genre;
    }

    public String getPublish_date() {
        return publish_date;
    }

    public void setPublish_date(String publish_date) {
        this.publish_date = publish_date;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    @Override
    public String toString() {
        return "Id: " + id + ", Title: " + title + ", Author: " + author + ", Genre: " + genre + ", Price: " + price
                + ", Description: " + description + ", publish_date: " + publish_date;
    }
}

File: Project/1dv600-lab-Node/app/dao/book.js
var book = (function () {
    function book(id, title, author, description, genre, price, publish_date) {
        this.id = id;
        this.title = title;
        this.author = author;
        this.price = price;
        this.genre = genre;
        this.publish_date = publish_date;
        this.description = description;
    }
    book.prototype.getId = function () {
        return this.id;
    };
    book.prototype.setId = function (id) {
        this.id = id;
    };
    book.prototype.getTitle = function () {
        return this.title;
    };
    book.prototype.setTitle = function (title) {
        this.title = title;
    };
    book.prototype.getAuthor = function () {
        return this.author;
    };
    book.prototype.setAuthor = function (author) {
        this.author = author;
    };
    book.prototype.getPrice = function () {
        return this.price;
    };
    book.prototype.setPrice = function (price) {
        this.price = price;
    };
    book.prototype.getGenre = function () {
        return this.genre;
    };
    book.prototype.setGenre = function (genre) {
        this.genre = genre;
    };
    book.prototype.getPublish_date = function () {
        return this.publish_date;
    };
    book.prototype.setPublish_date = function (publish_date) {
        this.publish_date = publish_date;
    };
    book.prototype.getDescription = function () {
        return this.description;
    };
    book.prototype.setDescription = function (description) {
        this.description = description;
    };
    book.prototype.toString = function () {
        return "Id: " + this.id + ", Title: " + this.title + ", Author: " + this.author + ", Genre: " + this.genre + ", Price: " + this.price + ", Description: " + this.description + ", publish_date: " + this.publish_date;
    };
    return book;
}());
